pointcloud_processing:
  ros__parameters:
    # depth_type: 32FC1
    tf_x: -0.052
    tf_y: 0.0
    tf_z: 0.025
    tf_roll: 0.0
    tf_pitch: 0.0
    tf_yaw: 0.0
    # tf_x: 0.042
    # tf_y: 0.0
    # tf_z: -0.045
    # tf_roll: 0
    # tf_pitch: 0
    # tf_yaw: 0
    # cam_fov_hor: 0.95668
    # cam_fov_ver: 0.74176
    # cam_width: 320
    # cam_height: 240



